.buildkite/pipeline.yml

```diff
@@ -7,14 +7,34 @@ steps:
     env:
       RAILS_ENV: test
     plugins:
-      - docker-compose#v3.0.0:
+      - docker-compose#v3.9.0:
           run: app
           env:
             - BUILDKITE_JOB_ID
             - CI
             - RAILS_ENV
             - BUILDKITE_ANALYTICS_TOKEN
 
+  - label: ":spider: muffet"
+    # We need to wait until rails has started before running muffet as otherwise it will error out
```

Comment:
May I suggest squashing all the commits into one and moving this comment to the commit message?

Reasoning behind this is: comments can become stale, while git log stays accurate with the current change

Reply:
I'm going to do both because if someone changes these lines, I think they are less likely to propagate an updated version of this message in their commit message than if the message were in a comment, them remembering to update the comment. This is especially so if they only make a cosmetic change to the lines. A comment would not be stale, but a commit message would be hidden behind the new commit.